src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,40 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.jsx';
 
-const ChatEntry = () => {
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, likeMessage, mainSenderColor, remoteSenderColor}) => {
+  const likebutton = liked ? '❤️' : '🤍';
+  const MainSender = 'Vladimir';
```

Comment:
```suggestion
  const MAIN_SENDER = 'Vladimir';
```
Constant variables should be named with all capital letters

Reply:
Notice this exact variable with the same exact value is repeated here (you also wrote it on line 11 of App.jsx). Could this value be passed as a prop from App.jsx to ChatEntry.jsx intead of repeating it here?